kiwi/schema/kiwi.rng

```diff
@@ -3890,7 +3890,130 @@ and setup the system disk.</a:documentation>
+          <value>ASERIES</value>
+          <value>HPNonStopOS</value>
+          <value>HPNonStopOSS</value>
+          <value>BS2000</value>
+          <value>LINUX</value>
+          <value>Lynx</value>
+          <value>XENIX</value>
+          <value>VM</value>
+          <value>InteractiveUNIX</value>
+          <value>BSDUNIX</value>
+          <value>FreeBSD</value>
+          <value>NetBSD</value>
+          <value>GNUHurd</value>
+          <value>OS9</value>
+          <value>MACHKernel</value>
+          <value>Inferno</value>
+          <value>QNX</value>
+          <value>EPOC</value>
+          <value>IxWorks</value>
+          <value>VxWorks</value>
+          <value>MiNT</value>
+          <value>BeOS</value>
+          <value>HPMPE</value>
+          <value>NextStep</value>
+          <value>PalmPilot</value>
+          <value>Rhapsody</value>
+          <value>Windows2000</value>
+          <value>Dedicated</value>
+          <value>OS390</value>
+          <value>VSE</value>
+          <value>TPF</value>
+          <value>WindowsMe</value>
+          <value>CalderaOpenUNIX</value>
+          <value>OpenBSD</value>
+          <value>NotApplicable</value>
+          <value>WindowsXP</value>
+          <value>zOS</value>
+          <value>MicrosoftWindowsServer2003</value>
+          <value>MicrosoftWindowsServer2003_64</value>
+          <value>WindowsXP_64</value>
+          <value>WindowsXPEmbedded</value>
+          <value>WindowsVista</value>
+          <value>WindowsVista_64</value>
+          <value>WindowsEmbeddedforPointofService</value>
+          <value>MicrosoftWindowsServer2008</value>
+          <value>MicrosoftWindowsServer2008_64</value>
+          <value>FreeBSD_64</value>
+          <value>RedHatEnterpriseLinux</value>
+          <value>RedHatEnterpriseLinux_64</value>
+          <value>Solaris_64</value>
+          <value>SUSE</value>
+          <value>SUSE_64</value>
+          <value>SLES</value>
+          <value>SLES_64</value>
+          <value>NovellOES</value>
+          <value>NovellLinuxDesktop</value>
+          <value>SunJavaDesktopSystem</value>
+          <value>Mandriva</value>
+          <value>Mandriva_64</value>
+          <value>TurboLinux</value>
+          <value>TurboLinux_64</value>
+          <value>Ubuntu</value>
+          <value>Ubuntu_64</value>
+          <value>Debian</value>
+          <value>Debian_64</value>
+          <value>Linux_2.4_x</value>
+          <value>Linux_2.4_x64</value>
+          <value>Linux_2.6_x</value>
+          <value>Linux_2.6_x64</value>
+          <value>Linux_64</value>
+          <value>Other_64</value>
+          <value>WindowsServer2008R2</value>
+          <value>VMwareESXi</value>
+          <value>Windows7</value>
+          <value>CentOS</value>
+          <value>CentOS_64</value>
+          <value>OracleEnterpriseLinux</value>
+          <value>OracleEnterpriseLinux_64</value>
+          <value>eComStation</value>
+        </choice>
```

Comment:
This list is problematic, since it changes from release to release of VirtualBox. Moreover, I think this list is actually incomplete, since distros patch in more values (Fedora and Mageia, for example, patch in identification for Mageia).

Reply:
As said in the PR description: I am really not sold on this and given your input, I agree that this is a bad idea.

Unfortunately, I don't really have a better one, beside using distro-detection code or using an universal fallback. I am also not too sure how much of a difference this ID actually makes, because packer gave my opensuse boxes the ID 94 (= Ubuntu x64).